dataland-frontend/src/components/forms/parts/fields/MostImportantProductsFormField.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <FormListFormField
+    :name="name"
+    :label="label"
+    :description="description"
+    :required="required"
+    :validation="validation"
+    :validation-label="validationLabel"
+    sub-form-component="ProductFormElement"
+    data-test-add-button="addNewProductButton"
+    label-add-button="ADD NEW Product"
+    data-test-sub-form="productSection"
```

Comment:
Is it intended that no prop for `dataTestRemoveButton` is passed?

Reply:
Yup, not needed in tests.

Reply:
👍